deploy/docker/Dockerfile

```diff
@@ -5,15 +5,8 @@ LABEL maintainer="info@cert.pl"
 RUN apk add --no-cache postgresql-client postgresql-dev libmagic
 
 COPY requirements.txt /tmp/requirements.txt
-RUN apk add --no-cache -t build libffi libffi-dev py3-cffi build-base python3-dev automake m4 perl autoconf libtool \
-    && wget -O /tmp/ssdeep.tar.gz https://github.com/ssdeep-project/ssdeep/releases/download/release-2.14.1/ssdeep-2.14.1.tar.gz \
-    && cd /tmp \
-    && tar -xopf /tmp/ssdeep.tar.gz \
-    && cd ssdeep-2.14.1 \
-    && ./configure \
-    && make \
-    && make install \
-    && cd /tmp && pip --no-cache-dir install -r requirements.txt \
+RUN apk add --no-cache -t build libffi libffi-dev py3-cffi build-base python3-dev automake m4 autoconf libtool \
+    && cd /tmp && BUILD_LIB=1 pip --no-cache-dir install -r requirements.txt \
```

Comment:
nice! do we still need `cd /tmp`?

Reply:
Nope, we should provide absolute path here. Fixed.